impeller/aiks/experimental_canvas.cc

```diff
@@ -670,6 +705,10 @@ void ExperimentalCanvas::DrawTextFrame(
 
 void ExperimentalCanvas::AddRenderEntityToCurrentPass(Entity entity,
                                                       bool reuse_depth) {
+  if (IsSkipping()) {
```

Comment:
Should we test this earlier? Before we make the entity in the first place?

Reply:
Eventually yes, as we won't ever make entities. But that logic is spread throughout the old and new canvas, so moving it earlier isn't easy as the old canvas doesn't use this check yet.